diff --git a/app/validators/status_length_validator.rb b/app/validators/status_length_validator.rb
index 575aaf1869..ecefca6fe9 100644
--- a/app/validators/status_length_validator.rb
+++ b/app/validators/status_length_validator.rb
@@ -1,7 +1,7 @@
 # frozen_string_literal: true

 class StatusLengthValidator < ActiveModel::Validator
-  MAX_CHARS = 500
+  MAX_CHARS = 5000
   URL_PLACEHOLDER_CHARS = 23
   URL_PLACEHOLDER = 'x' * 23

diff --git a/spec/validators/status_length_validator_spec.rb b/spec/validators/status_length_validator_spec.rb
index 050b7500bb..e5a2d6f013 100644
--- a/spec/validators/status_length_validator_spec.rb
+++ b/spec/validators/status_length_validator_spec.rb
@@ -4,7 +4,7 @@ require 'rails_helper'

 RSpec.describe StatusLengthValidator do
   describe '#validate' do
-    before { stub_const("#{described_class}::MAX_CHARS", 500) } # Example values below are relative to this baseline
+    before { stub_const("#{described_class}::MAX_CHARS", 5000) } # Example values below are relative to this baseline

     it 'does not add errors onto remote statuses' do
       status = instance_double(Status, local?: false)
